cmd/runner/main.go

```diff
@@ -1,13 +1,33 @@
+// Author: Neel Shah, 2020
+// main.go is a runner file.
+
 package main
 
 import (
 	"fmt"
 
+	"github.com/neelkshah/pandora/pkg/helper"
 	"github.com/neelkshah/pandora/pkg/linkedlist"
 )
 
 func main() {
-	fmt.Println("Hello")
-	linkedlist.Printer("Yay!")
-	linkedlist.Printer("Nay!")
+	var linkedListInstance = linkedlist.CreateLinkedList()
+	var nums = [][]uint64{{1, 435}, {2, 345}, {2, 672}, {4, 243}, {5, 2656}, {6, 6536}}
+	for _, iter := range nums {
+		linkedListInstance.Append(helper.IntToByte(iter[0]), helper.IntToByte(iter[1]))
+	}
+	linkedListInstance.Print()
+
+	fmt.Println("\n\nGET")
+
+	var arr, status = linkedListInstance.Get(helper.IntToByte(2))
+	var result = make([]uint64, 0)
+
+	for _, el := range arr {
+		result = append(result, helper.ByteToInt(el))
+	}
+
+	if status {
```

Comment:
Should be !status

Reply:
This file is just for quick testing, should not have pushed it to git. My bad, deleted.